### DISCUSSION TITLE
Three view for collections

### DISCUSSION THREAD

**Discussion by Matthijz98** [Thinkmill/keystatic#960] [Ideas]
Would be cool if nested content could be shown as a three.
Becouse now it is just a list:
![image](https://github.com/Thinkmill/keystatic/assets/17087153/fa50dae3-03cd-4d48-9390-1060d236a111)

**Comment by simonswiss** [comment]
Converting this to an idea in discussions — not sure how feasible it is at the moment, but that would definitely make a lot of sense for collections with a lot of nested content 🤙

**Comment by Matthijz98** [comment]
I thought about this some more.
What would be a cool option for this feature is drag and drop support not only for the nested level but also the order.
And it saves that orders as a number in the frontmatter and then that can be used when getting the data.
This would be really cool for collections that i want to use to build a nav structure based on.

I now have a separate file where i define a object that i use to build the nav for my site. But that file can not be edited trough keystatic. I could do that with singelton but the type checking would be really hard.
